src/CSFML/System/Time.cpp

```diff
@@ -56,14 +56,14 @@ int64_t sfTime_asMicroseconds(sfTime time)
 ////////////////////////////////////////////////////////////
 sfTime sfSeconds(float amount)
 {
-    return {static_cast<int64_t>(amount * 1000000)};
+    return {static_cast<int64_t>(amount) * 1000000};
```

Comment:
If I pass `1.52f`, it's going to be truncated to `1` with such a cast.

Reply:
haha I didn't realize that also float is definitely big enough to hold all values of uint64_t I should only change for the other function, thanks for noticing